developer_guides/virtualization/files/q-v6.sh

```diff
@@ -2,38 +2,38 @@
 # corresponds to the lower ethernet socket, "virbr0" is the bridge - the
 # master of both the physical ethernet interface and the virtual network.
 # "disk" is a partition, used to store a VM image, "path" is a mount point.
+# Note: this is an example file. You might want to adjust it. E.g. you might
+# not have access to a German keyboard, in which case you probably want to
+# remove the "-k de" qemu launch option. Also note, that it starts a second
+# instance of the QEMU VNC server, which then listens on port 5901.
```

Comment:
Technical parts are fine. @deb-intel I would probably rephrase this part ?

Reply:
Ok, no worries. I will edit it.